### ISSUE TITLE
[Suggestion]: Config to disable tractor beams 

### ISSUE THREAD

**Issue by TaintedBlockmObile** [nonamecrackers2/crackers-wither-storm-mod#1731]
### Description of feature request

This would disable the cluster pick up for the tractor beams.

**Comment by WitherBean** [comment]
You can just turn off the cluster pickup rate or at least just slow it down.

**Comment by TaintedBlockmObile** [comment]
Yeah but that still doesn't stop tractor beams from picking stuff up by chance

**Comment by WitherBean** [comment]
You could fix this with a datapack, but it would be a good idea for this in the mod by itself.
